tests/test_migrate.py

```diff
@@ -97,10 +109,48 @@ def test_multiple_vectors_collection(source_client, dest_client, request) -> Non
     """
     source_client: QdrantClient = request.getfixturevalue(source_client)
     dest_client: QdrantClient = request.getfixturevalue(dest_client)
+    vectors_config = multi_vector_config
     collection_name = "multiple_vectors_collection"
-    initialize_fixture_collection(source_client, collection_name="multiple_vectors_collection")
-    points = generate_fixtures(VECTOR_NUMBER)
+    initialize_fixture_collection(
+        source_client, collection_name=collection_name, vectors_config=vectors_config
+    )
+    multi_vectors_points = generate_multivector_fixtures()
+    points = multi_vectors_points
+    source_client.upload_points(collection_name, points, wait=True)
+    source_client.migrate(dest_client)
+    compare_collections(
+        source_client,
+        dest_client,
+        num_vectors=VECTOR_NUMBER,
+        collection_name=collection_name,
+    )
```

Comment:
this test is about a different thing

in this test we have several dense vectors in a collection, "text", "image" and "code".
it's not about multi vectors

dense vectors have a type of "List[float]"
multi vectors have a type of "List[List[float]]"
So it should be a separate test

Reply:
this test uses default fixtures, e.g. multi_vector_config, it produces the following vectors:
multi-text
multi-image
multi-code

we also allow users to have unnamed multi vectors (like in `test_single_vector_collection`), could you also add such a test please?